### PR TITLE
Add install.sh to install Chaos Mesh

### PR DESCRIPTION
**Pull request by cwen0**
### What problem does this PR solve? 
add a quick script to install kind local Kubernetes and chaos-mesh #192 
### What is changed and how does it work?
we can use `install.sh -l kind` to install `Helm` `Kind` `Kubernetes` and `Chaos Mesh` 

```
$ ./install.sh -h
This script is used to install chaos-mesh.
Before running this script, please ensure that:
* have installed docker if you run chaos-mesh in local.
USAGE:
    install.sh [FLAGS] [OPTIONS]
FLAGS:
    -h, --help              Prints help information
OPTIONS:
    -v, --version           Version of chaos-mesh, default value: latest
    -l, --local [kind]      Choose a way to run a local kubernetes cluster, supported value: kind,
                            If this value is not set and the Kubernetes is not installed, this script will exit with 1.
    -n, --name              Name of Kubernetes cluster, default value: kind
        --kind-version      Version of the Kind tool, default value: v0.7.0
        --node-num          The count of the cluster nodes,default value: 6
        --k8s-version       Version of the Kubernetes cluster,default value: v1.12.8
        --volume-num        The volumes number of each kubernetes node,default value: 9
        --helm-version      Version of the helm tool, default value: v2.16.1
        --release-name      Release name of chaos-mesh, default value: chaos-mesh
        --namespace         Namespace of chaos-mesh, default value: chaos-testing
```
### Check List 

Tests 

 - Manual test (add detailed scripts or steps below)

### CODE REVIEW

**Comment by Yisaer** [comment]
I think providing the way to install chaosmesh by several simple yaml files is helpful enough for users to get a first quick trial. We don't need to set up a local Kubernetes cluster.
Here are some examples:
https://knative.dev/docs/install/knative-with-any-k8s/#installing-knative
https://istio.io/docs/setup/install/helm/#option-1-install-with-helm-via-helm-template
https://cert-manager.io/docs/installation/kubernetes/#installing-with-regular-manifests

**Comment by cwen0** [comment]
@Yisaer Sometimes users do not have a Kubernetes cluster directly available. I think local Kubernetes is also needed for chaosmesh. we can use this scripts by the following commands

```
./install.sh # install chaos-mesh without local Kubernetes 
./install.sh --local kind # install chaos-mesh with a local kind Kubernetes 
``` 
We provides helm files to install chaosmesh and this scripts just to help users to simplify the installation steps， like: install helm / kind / local kubernetes

**Comment by cwen0** [comment]
I will pull a single pr to update README.md and document.

File: install.sh [hunk 1]
```diff
@@ -0,0 +1,481 @@
...
+    local kind_name="kind"
+    local kind_version="v0.7.0"
+    local node_num=6
+    local k8s_version="v1.12.8"
+    local volume_num=9
+    local helm_version="v2.16.1"
+    local release_name="chaos-mesh"
+    local namespace="chaos-testing"
+
+	while [[ $# -gt 0 ]]
```
Comment (zhouqiang-cl) [inline install.sh]: 5 space?

File: install.sh [hunk 2]
```diff
@@ -0,0 +1,481 @@
...
+		install_kind "${kind_version}"
+		install_kubernetes_by_kind "${kind_name}" "${k8s_version}" "${node_num}" "${volume_num}"
+	fi
+
+	install_chaos_mesh "${release_name}" "${namespace}" "${local-kube}"
+}
+
+prepare_env() {
+	mkdir -p ~/local/bin
+	export PTAH=~/local/bin:$PATH
```
Comment (zhouqiang-cl) [inline install.sh]: did we need to add into .bash_profile

**Comment by cwen0** [comment]
I will add `--force` flag to control whether to force reinstall chaos-mesh.

File: install.sh [hunk 3]
```diff
@@ -0,0 +1,481 @@
...
+Before running this script, please ensure that:
+* have installed docker if you run chaos-mesh in local.
+* have installed Kubernetes if you run chaos-mesh in normal Kubernetes cluster
+USAGE:
+    install.sh [FLAGS] [OPTIONS]
+FLAGS:
+    -h, --help              Prints help information
+OPTIONS:
+    -v, --version           Version of chaos-mesh, default value: latest
+    -l, --local [kind]      Choose a way to run a local kubernetes cluster, supported value: kind,
```
Comment (zhouqiang-cl) [inline install.sh]: Did we need to support in k8s(not local)

File: install.sh (hunk 2 above)
Comment (zhouqiang-cl) [reply install.sh]: @cwen0 PTAL

File: install.sh (hunk 3 above)
Comment (cwen0) [reply install.sh]: Yes, In this case, we just to check Kubernetes and install Chaos Mesh.

**Comment by cwen0** [comment]
@YangKeao @zhouqiang-cl @mahjonp PTAL again, thanks!

File: install.sh [hunk 4]
```diff
@@ -0,0 +1,553 @@
...
+    kubectl apply -f "${config_file}"
+}
+
+install_kind() {
+    local kind_version=$1
+    local force_install=$2
+
+    printf "Install Kind tool\n"
+
+	err_msg=$(kind version 2>&1 1>/dev/null)
```
Comment (mahjonp) [inline install.sh]: fmt

**Comment by cwen0** [comment]
@mahjonp @zhouqiang-cl PTAL again! thanks!

**Pull request merged**